sdk-base/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationPlugin.kt

```diff
@@ -3,12 +3,11 @@ package com.mapbox.maps.plugin.annotation
 import android.view.View
 import com.mapbox.maps.plugin.MapPlugin
 import com.mapbox.maps.plugin.MapSizePlugin
-import com.mapbox.maps.plugin.MapStyleObserverPlugin
 
 /**
  * Plugin interface for the annotation.
  */
-interface AnnotationPlugin : MapPlugin, MapSizePlugin, MapStyleObserverPlugin {
```

Comment:
If I recall correctly `MapStyleObserverPlugin` could be removed completely from codebase.

Reply:
em.. it could be useful in some cases, but yes, currently it is not used by any plugins..

Reply:
I guess we could always restore it if needed. If it's not used now I see no sense in leaving it in codebase.

Reply:
Removing this delegate could be a breaking change, wdyt @tobrun?